R/T_and_F_symbol_linter.R

```diff
@@ -50,8 +50,8 @@ T_and_F_symbol_linter <- function() { # nolint: object_name.
           sprintf(fmt, replacement_map[[symbol]], symbol)
         },
         type = "style",
-        column_number_xpath = "number(./@col2 + 1)",
-        range_end_xpath = "number(./@col2 + 1)"
+        column_number_xpath = "number(./@col2 + 1)", # mark at end
+        range_end_xpath = "number(./@col2 + 1)" # end after T/F for easy fixing
```

Comment:
case where it's trivial (bump by 1) doesn't need a note I'd say. but I'm fine either way

Reply:
I find the reason why we do it interesting enough to document.